lib/api/settings.js

```diff
@@ -0,0 +1,39 @@
+const Request = require('./request')
+
+class Settings extends Request {
+
+  constructor(httpClient){
+    super(httpClient)
+    this.httpClient = httpClient
+  }
+
+  get(channelId, accessToken) {
+    const headers = this.createHeaders(accessToken)
+    return this.httpClient
+      .get(`/v1/channels/${channelId}/settings`, {headers})
+      .then(response => response.data)
+  }
+
+  set(channelId, option, accessToken) {
```

Comment:
set → update, přijde mi to jasnější, když si představím, že se tam i maže

Reply:
Přejmenujeme `option` na `setting`?

Reply:
setting ok, set ~= update ve smyslu rewrite ok ale porad ze slova nevim jestli lze resource smazat